src/snmalloc/mem/corealloc.h

```diff
@@ -566,6 +569,12 @@ namespace snmalloc
       // This must occur before any freelists are constructed.
       entropy.init<typename Config::Pal>();
 
+      if (spare.length != 0)
+      {
+        Config::Backend::dealloc_meta_data(
```

Comment:
It might not hurt to have a comment here, for mere mortals like me that don't always readily recall that "backend" things come in per-frontend and shared state flavors.  Maybe something like...

```
/*
 * Seed this frontend's private metadata allocation cache with any
 * excess space from the metadata allocation holding the frontend
 * Allocator object itself.  This alleviates thundering herd
 * contention on the backend during startup: each slab opened now
 * makes one trip to the backend, for the slab itself, rather than
 * two, for the slab and its metadata.
 */
```